From c9330dc1d26123e78b75c773134825bd5a1f4a55 Mon Sep 17 00:00:00 2001
From: D치vid H치zi <david.hazi@arm.com>
Date: Thu, 7 Mar 2024 09:40:06 +0100
Subject: [PATCH] provisioning: Change provisioning log messages
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

* Corstone-315 has got a new provisioning method.
  To match the new method to the older platform,
  we change the last provisioning step's log
  messages, in this case the CI pass conditions
  can be unified.

Signed-off-by: D치vid H치zi <david.hazi@arm.com>
---
 .../arm/mps4/corstone315/provisioning/bl1_provisioning.c      | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/platform/ext/target/arm/mps4/corstone315/provisioning/bl1_provisioning.c b/platform/ext/target/arm/mps4/corstone315/provisioning/bl1_provisioning.c
index 61d0f1aaa..71ac2a1cd 100644
--- a/platform/ext/target/arm/mps4/corstone315/provisioning/bl1_provisioning.c
+++ b/platform/ext/target/arm/mps4/corstone315/provisioning/bl1_provisioning.c
@@ -107,7 +107,7 @@ static enum tfm_plat_err_t provision_psa_rot(void)
            (void *)&dm_bundle->values,
            PROVISIONING_BUNDLE_VALUES_SIZE + PROVISIONING_BUNDLE_DATA_SIZE);

-    BL1_LOG("[INF] Running DM provisioning bundle\r\n");
+    BL1_LOG("[INF] Running provisioning bundle\r\n");
     err = ((enum tfm_plat_err_t (*)(void))(PROVISIONING_BUNDLE_CODE_START | 0b1))();

     memset((void *)PROVISIONING_BUNDLE_CODE_START, 0,
@@ -182,7 +182,7 @@ enum tfm_plat_err_t tfm_plat_provisioning_perform(void)
     }
     if (lcs == LCM_LCS_DM) {

-        BL1_LOG("[INF] Waiting for DM provisioning bundle\r\n");
+        BL1_LOG("[INF] Waiting for provisioning bundle\r\n");
         while (dm_bundle->magic != DM_BUNDLE_MAGIC ||
                dm_bundle->magic2 != DM_BUNDLE_MAGIC) {
         }
--
2.40.1

